CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Fix `ManagementClusterAppFailed` and `WorkloadClusterAppFailed` to page balo only for the components they own
```

Comment:
Is `balo` `=` `ba(tman) + (ha)lo`?

Reply:
Yep

Reply:
This is confusing IMO, I would like to clearly state halo and batman instead for clearer understanding from everyone.

```suggestion
- Fix `ManagementClusterAppFailed` and `WorkloadClusterAppFailed` to page batman and halo only for the components they own
```